### PR TITLE
Runtime 10 table updates

### PR DESCRIPTION
**Pull request by lzdanski**
Add v10 info to reference tables

- https://astronomer.slack.com/archives/C015V2JFKT5/p1703665839579119

### CODE REVIEW

File: astro/runtime-image-architecture.md [hunk 1]
```diff
@@ -160,6 +162,7 @@ The following table lists the operating systems and architectures supported by e
 | 7             | Debian 11.3 (bullseye) | AMD64 and ARM64 |
 | 8             | Debian 11.3 (bullseye) | AMD64 and ARM64 |
 | 9             | Debian 11.3 (bullseye) | AMD64 and ARM64 |
+| 10            | Debian 11.3 (bullseye) | AMD64 and ARM64 |
```
Comment (BasPH) [inline astro/runtime-image-architecture.md:165]: I ran

```bash
docker run --rm -ti quay.io/astronomer/astro-runtime:10.0.0 bash
cat /etc/debian_version
```

Which gives Debian version 11.8 (also called bullseye)

File: software/runtime-image-architecture.md [hunk 2]
```diff
@@ -141,6 +143,7 @@ The following table lists the operating systems and architectures supported by e
 | 7             | Debian 11.3 (bullseye) | AMD64 and ARM64 |
 | 8             | Debian 11.3 (bullseye) | AMD64 and ARM64 |
 | 9             | Debian 11.3 (bullseye) | AMD64 and ARM64 |
+| 10            | Debian 11.3 (bullseye) | AMD64 and ARM64 |
```
Comment (BasPH) [inline software/runtime-image-architecture.md:146]: Same comment here

File: astro/runtime-image-architecture.md (hunk 1 above)
Comment (BasPH) [reply astro/runtime-image-architecture.md:165]: Just to be sure I checked the Debian versions on all Astro Runtime (AR) versions:

- AR 4: 11.1
- AR 5: 11.3
- AR 6: 11.5
- AR 7: 11.5
- AR 8: 11.7
- AR 9: 11.7
- AR 10: 11.8

(All called bullseye)

**Review by BasPH — APPROVED** [review]

**Comment by lzdanski** [comment]
Confirmed correct formatting in preview.

**Pull request merged**